### ISSUE TITLE
Fish 3.0.2 has indent issues on MAC and WSL2

### ISSUE THREAD

**Issue by marcusm** [oh-my-fish/theme-budspencer#60]
This only occurs for the budspencer theme. My other themes work fine.

 2  ✔  cd ~                                                              
Fish has been upgraded, and the scripts on your system are not compatible
with this prior instance of fish. You can probably run:

 exec fish
… to replace this process with a new one in-place.
- (line 1): function: Unexpected positional argument 'alias fish_indent=/usr/local/Cellar/fish/3.0.2/bin/fish_indent'
function fish_indent --wraps  --description 'alias fish_indent=/usr/local/Cellar/fish/3.0.2/bin/fish_indent';  /usr/local/Cellar/fish/3.0.2/bin/fish_indent  $argv; end
^
from sourcing file -
	called on line 72 of file /usr/local/Cellar/fish/3.0.2/share/fish/functions/alias.fish

in function 'alias'
	called on line 7 of file /usr/local/Cellar/fish/3.0.2/share/fish/functions/fish_indent.fish
	with parameter list 'fish_indent=/usr/local/Cellar/fish/3.0.2/bin/fish_indent'

from sourcing file /usr/local/Cellar/fish/3.0.2/share/fish/functions/fish_indent.fish
	called on line 263 of file ~/.config/fish/functions/fish_prompt.fish

in command substitution
	called on line 263 of file ~/.config/fish/functions/fish_prompt.fish

in command substitution
	called on line 527 of file ~/.config/fish/functions/fish_prompt.fish

in function '__budspencer_create_cmd_hist'
	called on standard input

in event handler: handler for generic event 'fish_prompt'

 1  ✔  omf doctor                                                                                           ~
Oh My Fish version:   6
OS type:              Darwin
Fish version:         fish, version 3.0.2
Git version:          git version 2.23.0
Git core.autocrlf:    no
Your shell is ready to swim.

**Comment by clayauld** [comment]
I've seen this error but only after an upgrade to fish or any components I had installed. In my case running `exec fish` fixed the issue. Is this the error you're seeing when you start a new terminal session?

**Comment by Shub1427** [comment]
I didn't want to open a new issue, since my issue resembles something what the title states here.

<img width="1039" alt="Screenshot 2019-11-08 at 2 47 21 PM" src="https://user-images.githubusercontent.com/11786283/68465063-bb79cd00-0237-11ea-8fe3-13039a20b04c.png">

As you can see above, too much padding inbetween and to the extreme right. Also, single letter type (first keypress) causes the above issue, where the right prompt looks really awkward.

<img width="1035" alt="Screenshot 2019-11-08 at 2 58 31 PM" src="https://user-images.githubusercontent.com/11786283/68465322-45299a80-0238-11ea-83e1-7d61b9319ae9.png">

**Comment by Shub1427** [comment]
`omf doctor`

```
Oh My Fish version:   6
OS type:              Darwin
Fish version:         fish, version 3.0.2
Git version:          git version 2.24.0
Git core.autocrlf:    no
Your shell is ready to swim.
```

Terminal used is Alacritty, (issue doesn't occurs on MacOS Terminal app) but I don't think it's Alacritty issue.

**Comment by clayauld** [comment]
I don't see any issue like that in iTerm. Can you attempt to use iTerm to see how that behaves?

**Comment by Shub1427** [comment]
Yeah! It works in Terminal.app/iTerm.app as well. Would be Alacritty issue then. Thanks @clayauld

**Comment by clayauld** [comment]
Yes, the issue is likely with how Alacritty renders the theme. I'm not sure at all how that terminal emulator works, so I wouldn't be able to help you unless they found a fix and created a PR for it.
